aws-datastore/src/androidTest/java/com/amplifyframework/datastore/storage/sqlite/SQLiteStorageAdapterClearTest.java

```diff
@@ -0,0 +1,111 @@
+/*
+ * Copyright 2019 Amazon.com, Inc. or its affiliates. All Rights Reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License").
+ * You may not use this file except in compliance with the License.
+ * A copy of the License is located at
+ *
+ *  http://aws.amazon.com/apache2.0
+ *
+ * or in the "license" file accompanying this file. This file is distributed
+ * on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either
+ * express or implied. See the License for the specific language governing
+ * permissions and limitations under the License.
+ */
+
+package com.amplifyframework.datastore.storage.sqlite;
+
+import android.content.Context;
+import androidx.test.core.app.ApplicationProvider;
+
+import com.amplifyframework.datastore.DataStoreException;
+import com.amplifyframework.datastore.StrictMode;
+import com.amplifyframework.datastore.storage.StorageItemChange;
+import com.amplifyframework.datastore.storage.SynchronousStorageAdapter;
+import com.amplifyframework.testmodels.commentsblog.AmplifyModelProvider;
+import com.amplifyframework.testmodels.commentsblog.BlogOwner;
+
+import org.junit.After;
+import org.junit.Before;
+import org.junit.BeforeClass;
+import org.junit.Test;
+
+import java.util.List;
+
+import io.reactivex.observers.TestObserver;
+
+import static com.amplifyframework.datastore.storage.sqlite.SQLiteStorageAdapter.DATABASE_NAME;
+import static org.junit.Assert.assertEquals;
+import static org.junit.Assert.assertFalse;
+import static org.junit.Assert.assertTrue;
+
+public class SQLiteStorageAdapterClearTest {
+    private SynchronousStorageAdapter adapter;
+    private Context context;
+    private TestObserver<StorageItemChange.Record> observer;
+
+    @BeforeClass
+    public static void enableStrictMode() {
+        StrictMode.enable();
+    }
+
+    @Before
+    public void setup() {
+        TestStorageAdapter.cleanup();
+        context = (ApplicationProvider.getApplicationContext());
+        adapter = TestStorageAdapter.create(AmplifyModelProvider.getInstance());
+        observer = TestObserver.create();
+        adapter.observe().subscribe(observer);
```

Comment:
There's a short-hand for this, [`Observable#test()`](http://reactivex.io/RxJava/javadoc/rx/Observable.html#test--):

```
TestObserver<StorageItemChange.Record> observer = adapter.observe().test();
```

It only saves a line there, but it doesn't need to be disposed, which is neat. And it's _slightly_ more expressive of the intention.

Reply:
Updating this one. I had made the change locally then stashed it and ended up getting lost.